docs/writing-stories/args.md

```diff
@@ -98,6 +98,20 @@ Args are useful when writing stories for composite components that are assembled
 
 <!-- prettier-ignore-end -->
 
+## Args can modify any aspect of your components
```

Comment:
Can we make the prose more framework agnostic? This text gets displayed to all frameworks (only the code samples change). 

Also, how would this work in React?

Reply:
I realize that, but I don't know enough about React to know how you'd use args other than as props.  And that was the problem that I was trying to address: frameworks that have concepts like slots that don't exist in React (AFAIK)

Reply:
Couldn't React be done like this?

```jsx
const Template = (args, { argTypes }) => (
     <page {...args}>
       {args.footer}
     </page>
);

 export const CustomFooter = Template.bind({});
 CustomFooter.args = {
   footer: <a href="https://storybook.js.org/">Built with Storybook</a>,
 };
```

A little sloppy, but I think that is the equivalent "args as a child" concept. How can we frame those both in this?

Reply:
If there isn't a snippet for every language, what do I do? Do I copy the React variant to the `common` and all the other supported frameworks?

Reply:
Arg values have to be telejson serializable @phated so that example won't work. Hopefully we can mitigate that limitation with #13888

Reply:
Would this (simpler) example be valid for React? Do the examples have to do exactly the same thing for all frameworks? 

```
const Template = (args, { argTypes }) => (
     <page {...args}>
       <footer>{args.footer}</footer
     </page>
);

 export const CustomFooter = Template.bind({});
 CustomFooter.args = {
   footer: 'Built with Storybook',
 };
 ```

Reply:
The examples have to do approximately the same thing. 

The code will necessarily be different but the technique should be similar. The code snippet for each framework can be as verbose as we need to accomplish the technique.

Reply:
@domyen If there isn't a snippet for every framework, what do I do? 
1. Do I copy the React version to the common? 
2. Copy React version to and all the other supported frameworks?

Reply:
@saschwarz Add snippets for Vue and React 🙏 

All the other frameworks will automatically be shown React until someone contributes the code sample. There's a call to action to do this built into the snippet component already.